Driverless AI/natural-language-processing-tutorial-sentiment-analysis /.ipynb_checkpoints/natural-language-processing-tutorial-sentiment-analysis-checkpoint.md

```diff
@@ -517,7 +506,7 @@ This option gives a brief summary of the entire experiment including :
 
 There are also several plots adjacent to the summary tab that give insight into the experiment. If you are interested in learning more about each plot and the metrics derived from those plots covered in this section, then check out our next tutorial [Machine Learning Experiment Scoring and Analysis Tutorial - Financial Focus](https://h2oai.github.io/tutorials/machine-learning-experiment-scoring-and-analysis-tutorial-financial-focus/#0).
```

Comment:
@Castro38 should be "check out this tutorial" and not "then check out our next tutorial"

Reply:
@5675sp Done